src/Utils/ConfigurationManager.php

```diff
@@ -119,18 +119,19 @@ public function readTarget($target)
             $environment = $exploded[1];
         }
 
-        if (!array_key_exists($site, $this->sites)) {
+        $sites = $this->getSites();
```

Comment:
By calling the `$this->getSites();`method the `this->sites` array wiill be filled with data.

Reply:
@jmolivas a proposal by nvaken here https://github.com/hechoendrupal/drupal-console/issues/3761 to have a single point of access to sites property. 
Theoretically, the `$this->site` can be empty, if we didn't call `$this->getSite()` before in https://github.com/hechoendrupal/drupal-console-core/blob/master/src/Command/Debug/SiteCommand.php#L70

Reply:
@LOBsTerr  What I meaning is to add the ` $this->getSites();` above this line:
```
if (!array_key_exists($site, $this->sites)) {
```

Reply:
So what you’re saying is call $this->getSites() without assigning it to $sites and just keep getting data from $this->sites directly? I think assigning it to a local variable (like @lobsterr did) is best practice performance wise, isn’t it?

Reply:
@nvaken exactly but is true local scope var is faster than the class one, we can keep this PR as it is.

Reply:
I will go ahead and merge it

Reply:
Awesome!👏👏